source/Random.cpp

```diff
@@ -112,3 +116,36 @@ double Random::Normal()
 #endif
 	return normal(gen);
 }
+
+
+
+// Get a random UUID
```

Comment:
Maybe add some text to the comment here how an UUID looks?
Also, comments in ES need to be full sentences and end with a dot.

Reply:
I added an example, the IETF RFC number, and an explanation to both Random.cpp and Random.h.